h/models/helpers.py

```diff
@@ -0,0 +1,6 @@
+def repr_(obj, attrs):
+    class_name = type(obj).__name__
+    attrs = {attrname: getattr(obj, attrname) for attrname in attrs}
+    return (
+        f"{class_name}({', '.join(f'{name}={value!r}' for name, value in attrs.items())})"
+    )
```

Comment:
Used this code in a couple of places now so I moved it to a helper function.

There are lots of `__repr__()` implementations in h which could probably be better if they were changed to use this function. Can do that in future maybe.

Reply:
If we updated h to use [SQLAlchemy's dataclasses support](https://docs.sqlalchemy.org/en/20/orm/dataclasses.html) (which we already use in Via) then we wouldn't need this.